### ISSUE TITLE
[Bug Report][3.3.10] VRatingItem button missing aria-label

### ISSUE THREAD

**Issue by bogdanciuca** [vuetifyjs/vuetify#17903]
### Environment
**Vuetify Version:** 3.3.10
**Vue Version:** 3.3.4
**Browsers:** Chrome 114.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Use developer tools to inspect markup of `VRatingItem`.

### Expected Behavior
Button's `aria-label` should be set to `itemAriaLabel` prop value.

### Actual Behavior
Button doesn't have `aria-label` set.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlkM9OwzAMh1/FymUgLQ1ocJk2NJ5j2aFrPQg0f5R4RWjqu+O0Raq6y6Qd4/z8fbb3F/EeQtGeUazFhtCGpiR80w5g08pYknEf0Erra2y2WgwFLUBxZKMmeX6mKppAkJDOoScYG3wkuEDEE3Rwit7CglWLHAeovEsEo2ObQw+vjxk7gBghlmJgSFuG4it5x1Necq8eP5IWa+grucbs/NbikyiktVJV7biNZzdtLByScsGqHcdUPDsyFmXt7W5VrIoXVZtE03KBycpj9D8JI0O0WE40ZE6/N6jGZG94fhoUY0025TFlxxVbsbDFKCO6GiPGW1eatU3Xmn1drZbtnXYdH7wx7jvNbl2l/s77/yHv27mnZdQhG7ul6Adl9dAhDn9SZ+A2)

**Comment by websitevirtuoso** [comment]
1. You gave wrong palyground.
2.
```
<template>
  <v-app>
    <v-rating item-aria-label="label area" v-model="rating" />
  </v-app>
</template>

<script setup>
  import { ref } from 'vue'

  const rating = ref(5)
</script>
```

This prop set correct label
![image](https://github.com/vuetifyjs/vuetify/assets/5112918/1eae7814-5b6a-4cec-ae17-c6c466101c55)

But this tag is hidden. DO you expect it visible?

I am just trying to make it clear

**Comment by bogdanciuca** [comment]
> 1. You gave wrong palyground.
What do you mean?
>2 
I am referring to `button[aria-label]`, not the `label`'s text.

**Comment by websitevirtuoso** [comment]
but we have many buttons should they have all the same values?

We have ability to set multiple labels
like

      :item-labels="['sad', '', '', '', 'happy']"
      item-label-position="top"

**Comment by bogdanciuca** [comment]
You don't need to specify the `item-aria-label`, it already has a default value, i.e.
```
itemAriaLabel: {
    type: String,
    default: '$vuetify.rating.ariaLabel.item',
}
```
Which uses localized values, e.g.
```
rating: {
    ariaLabel: {
      item: 'Rating {0} of {1}',
    },
}
```

**Comment by websitevirtuoso** [comment]
Oh I see.
![image](https://github.com/vuetifyjs/vuetify/assets/5112918/781059c9-da91-4860-95e3-468c3208ad39)

SO we expect for better accessibility set attr "aria-label" to button is it right?
Did you check it via any reader?

**Comment by bogdanciuca** [comment]
> SO we expect for better accessibility set attr "aria-label" to button is it right?

Exactly.

> Did you check it via any reader?

No, but it fails the Accesibility test on [PageSpeed Insights](https://pagespeed.web.dev/)

**Comment by websitevirtuoso** [comment]
Ok. I will try to help. but not now. when will have a little time. I will create PR. Thanks

**Comment by bogdanciuca** [comment]
I already created the PR, it links to this issue. See #17904

**Comment by websitevirtuoso** [comment]
awesome
